doc/schema/libdoc_schema.json

```diff
@@ -184,6 +194,93 @@
           "type": "string"
         }
       }
+    },
+    "enumType": {
+      "$id": "#/definitions/enumType",
+      "type": "object",
+      "title": "Enum DataType schema",
+      "required": [
+        "name",
+        "super",
+        "doc",
+        "members"
+      ],
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "super": {
+          "type": "string"
+        },
+        "doc": {
+          "type": "string"
+        },
+        "members": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/enum_member"
+          },
+          "uniqueItems": true
+        }
+      }
+    },
+    "enum_member": {
+      "$id": "#/definitions/enum_member",
+      "type": "object",
+      "title": "Enum Member schema",
+      "required": [
+        "name",
+        "value"
+      ],
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "typedDictType": {
+      "$id": "#/definitions/typedDictType",
+      "type": "object",
+      "title": "TypedDict Type schema",
+      "required": [
+        "name",
+        "super",
+        "doc",
+        "items",
+        "required_keys",
+        "optional_keys"
+      ],
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "super": {
+          "type": "string"
+        },
+        "doc": {
+          "type": "string"
+        },
+        "items": {
+          "type": "object"
+        },
+        "required_keys": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "uniqueItems": true
+        },
+        "optional_keys": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "uniqueItems": true
+        }
+      }
```

Comment:
Seems to be related to data types and not to this PR. That enhancement is coming soon so no biggie.